### ISSUE TITLE
require collection licenses

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8131]
Help us understand your request (check below):

- [ ] other

**Describe what you're trying to do** 

https://github.com/ArctosDB/internal/issues/328

I'm working on the initial collection creation request, and surprised to find that licenses are not included. I can't think of any reason (other than being ignored because it wasn't required!) that a collection would not want licenses. Can we make this a requirement? 

I don't think anyone cares what the terms of the license are, and this should not be seen as an attempt to influence that.  We currently have [do what you want](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctdata_license#cc0), and anything up to and including 'talk to our lawyers before even reading this' is possible, I just don't like the option of "we're not saying, you figure it out!"

Here are the 177 (!!) collections missing at least one license.

[temp_no_lincense.csv](https://github.com/user-attachments/files/17105074/temp_no_lincense.csv)

**Comment by ewommack** [comment]
I think some of the aggregators require us to have certain licenses. @dbloom do I have that right?

**Comment by dustymc** [comment]
>aggregators require

Yes-but-no. GBIF can't **require** anything of any collection, but they can (and do) refuse to publish data which doesn't fall under one of their acceptable licenses. Certainly a consideration, not a requirement.

**Comment by campmlc** [comment]
This definitely needs to happen in collection onboarding, and we need to clearly explain the following: 
https://ipt.gbif.org/manual/en/ipt/latest/applying-license
In accordance with the [GBIF licensing policy](http://www.gbif.org/terms/licences), a dataset should be made available for use under CC0, CC-BY, or CC-BY-NC. GBIF’s recommended best practice is to use the most recent version, which is 4.0 for CC-BY and CC-BY-NC. This is in line with Creative Commons' recommendation that:

CC0 : under which data are made available for any use without restriction
CC BY : under which data are made available for any use provided that attribution is appropriately given for the sources of data used, in the manner specified by the owner
CC BY-NC : under which data are made available for any use provided that attribution is appropriately given and provided the use is not for commercial purposes

**Comment by dustymc** [comment]
@campmlc I edited your reply, one step at a time please!

**Comment by campmlc** [comment]
I would say we need to make this a requirement for collections that intend to publish to GBIF. They may not know what the consequences are of not meeting this requirement, but we do, and we owe it to incoming collections to make this an explicit step.

**Comment by dustymc** [comment]
>for collections that intend to publish to GBIF

I don't think that's relevant to this discussion. (There's an effort to simplify such things running in the background, but "doesn't have a license, can't share with someProject" and "has a license that's not compatible with someProject" are ~same for both me and someone who might want to share with someProject. That - both big-picture and specific use cases, and extending to terms - needs documentation and direction, but I don't think it needs to distract from the simple question of whether we can/should require licenses. In case it's not clear, I think we should require licenses, and I think it'll be a mostly-unnoticeable part of creating a collection - but maybe not as painless for the 177 collections, including 3 MSB, listed above.)

**Comment by campmlc** [comment]
Then I agree. We should require licenses. Solves many different problems, including the one I raised as well as the 177 collections that are likely not even aware they don't have one.

**Comment by dustymc** [comment]
AWG discussion.

Everybody likes requiring license.

Officers will help update existing collections (ping me for data).

DLM and LKV will begin "requiring" this for new collections.

**Comment by campmlc** [comment]
pinging @dustymc for data on which collections lack licenses.

**Comment by dustymc** [comment]
@campmlc there are data in the first comment.

**Comment by dustymc** [comment]
Chat with @mkoo 

* manage collection: add a big red "pick a license!" box
* End of year: we will assign a license to any collection which still does not have one

**Comment by dustymc** [comment]
Dev issue: https://github.com/ArctosDB/dev/issues/78

Added to calendar for January 13 (arbitrary "near end of year" date when someone will probably be around, can be changed as necessary).

For AWG meeting:
* select default internal license (default-default: https://arctos.database.museum/info/ctDocumentation.cfm?table=ctdata_license#cc_by_nc_sa) 
* select default external license (default-default: https://arctos.database.museum/info/ctDocumentation.cfm?table=ctdata_license#cc_by_nc_sa) 
* announce to community, stress that 
    * any license is fine, this isn't limited to current options or CC or anything else, just [ask](https://github.com/ArctosDB/arctos/issues/new?assignees=&labels=&projects=&template=request.md&title=Request+-+) if you need a new thing, and
    * PLEASE just update your collections so we don't have to actually use these 'defaults'
* (DLM request/addition) - ask for volunteers to review and update (or supplement) [documentation](https://handbook.arctosdb.org/documentation/collection.html) as necessary

**Comment by dustymc** [comment]
Discussed https://docs.google.com/document/d/1N9mx7q0UHLgpNvIgdoRjLCqidqYzS0d_yuHi1tPVSCY/edit?tab=t.0

@ewommack was there some plan to further communicate the impending requirement?

Fresh data: 

[temp_no_license.csv.zip](https://github.com/user-attachments/files/17513799/temp_no_license.csv.zip)

**Comment by ewommack** [comment]
Not that I saw at AWG, but perhaps I missed something? I didn't see a call for a newsletter or email.

**Comment by dustymc** [comment]
@mkoo ping - do we want an announcement re: [January](https://github.com/ArctosDB/arctos/issues/8131#issuecomment-2414505101)

**Comment by mkoo** [comment]
I'm wrapping this info in my email regarding collection GUID prefix Exhibit 3 and have started contacting collections directly.
Transferring to Internal to track the task and will close once all 60 members are contacted.
